app/client/app/components/suitability-form/suitability-form.component.ts

```diff
@@ -222,36 +227,36 @@ export class SuitabilityFormComponent {
         if (this.totalPoints < 18) { this.warning = true; }
     }
 
-    next(event, nextSection) {
-        switch (nextSection) {
-            case 'section2':
-                this.showSectionBtn2 = true;
-                this.clicked(nextSection);
-                break;
-            case 'section3':
-                this.showSectionBtn3 = true;
-                this.clicked(nextSection);
-                break;
-            case 'section4':
-                this.showSectionBtn4 = true;
-                this.clicked(nextSection);
-                break;
-            case 'section5':
-                this.showSectionBtn5 = true;
-                this.clicked(nextSection);
-                break;
-            case 'section6':
-                this.showSectionBtn6 = true;
-                this.clicked(nextSection);
-                break;
-            default:
-        }
-    }
+    // next(event, nextSection) {
+    //     switch (nextSection) {
```

Comment:
It's ok to delete unused code... If you think you might need it later and will forget the commit that included it then create a stash/branch and leave the head commit pointing to the state you want to preserve.